Modules/AwardDKP.lua

```diff
@@ -0,0 +1,105 @@
+local _, core = ...;
+local _G = _G;
+local MonDKP = core.MonDKP;
+local L = core.L;
+
+local function GetEligibleGuildMembers(onlineOnly, sameZone, currZone)
+    -- Get list of guild members, optionally filtered for only online members and/or members currently in zone currZone
+    -- Returns table playerTable where playerTable[playerName] is not nil iff playerName is eligible
+    local playerTable = {}
+    for playerIndex = 1, GetNumGuildMembers() do
+        local name, _, _, _, _, zone, _, _, online, _, _, _, _, _, _, _ = GetGuildRosterInfo(playerIndex);
+        if ((not onlineOnly) or online) and ((not sameZone) or (zone == currZone)) then
+            playerTable[name] = 1
+        end
+    end
+    return playerTable
+end
+
+function MonDKP:AwardPlayer(name, amount)
+	local search = MonDKP:Table_Search(MonDKP_DKPTable, name, "player")
+	local path;
```

Comment:
`path` is a bit confisuing name in this context :)

Reply:
Changed to ```player``` - btw this was old code I just put to a different location, not by me ;)